.github/workflows/packaging.yml

```diff
@@ -0,0 +1,54 @@
+name: Packaging release assets
+
+on:
+  push:
+    branches:
+    - 'pre-release*'
+    - 'prerelease*'
+    - 'release-*'
+
+jobs:
+  build-deb:
+    name: Build DEB package
+    runs-on: ubuntu-latest
+    container: debian:jessie
+    steps:
+    - name: Install tools
+      run: apt-get update && apt-get install --yes patch unzip git gcc make curl pkg-config
+    - name: Checkout r2
+      run: |
+        git clone https://github.com/${{ github.repository }}
+        cd radare2
+        git fetch origin ${{ github.ref }}
+        git checkout -b local_branch FETCH_HEAD
+    - name: Making the static build
+      run: sys/static.sh
+      working-directory: radare2
+    - name: Cleaning up the unnecessary files
+      run: git clean -xdf
+      working-directory: radare2
+    - name: Preparing the Debian package
+      run: sys/debian.sh
+      working-directory: radare2
+    - name: Create Release
```

Comment:
Do you know what happens when we will have multiple jobs that do "Create release"? I'm thinking about jobs for uploading cydia package, android apk, etc. Will it generate multiple releases or re-use the existing one?

Reply:
Ops sorry, ignore this comment.